### PR TITLE
Dockerized extractor

### PR DESCRIPTION
**Pull request by phretor**
Due to some tools being hard if not impossible to install on non-Linux systems, I've created this Docker-based wrapper.

I tested it successfully on a Mac running macOS Sierra.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+RUN \
+  adduser --disabled-password \
+          --gecos '' \
+          --home /home/extractor \
+          extractor
+
+USER extractor
+WORKDIR /home/extractor
+
+RUN git clone https://github.com/phretor/extractor.git
```
Comment (ddcc) [inline Dockerfile]: Could you change this to point to this repository?

File: README.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ Firmware images with multiple filesystems are not fully supported; this tool
 cannot reassemble them and will instead extract the first filesystem that has
 sufficient UNIX-like root directories (e.g. `/bin`, `/etc/`, etc.)
 
+For the impatients: Dockerize all the things!
+=============================================
+1. Install [Docker](https://docs.docker.com/engine/getstarted/)
+2. Run the dockerized extractor
+```
+git clone https://github.com/phretor/extractor.git
```
Comment (ddcc) [inline README.md]: Likewise

File: Dockerfile (hunk 1 above)
Comment (phretor) [inline Dockerfile]: Sure thing!

File: README.md (hunk 2 above)
Comment (phretor) [inline README.md]: Sure thing!

**Comment by phretor** [comment]
Done. You should be able to merge @ddcc

**Comment by ddcc** [comment]
How is your docker base image set up? Is it any different from a stock Ubuntu image ?

**Comment by ddcc** [comment]
That commit wasn't supposed to go to your fork, I'm surprised it actually let me push it. Unfortunately, I can't remove it and force-push to roll back.

**Comment by ddcc** [comment]
Thanks for the pull request! Just wanted to let you know that I've made some other changes and rebuilt the base image.

**Comment by phretor** [comment]
Dominic: are your changes already reflected in the Dockerfile?

**Comment by ddcc** [comment]
Yes. The Dockerfile itself wasn't changed, the changes were to this repository, but I couldn't update the git copy in the image without rebuilding the whole thing.

**Comment by phretor** [comment]
Oh I see, you changed the tempfs thing: good idea. New image v0.2 on the hub!

**Pull request merged**